assets/src/edit-story/elements/image/display.js

```diff
@@ -53,6 +53,7 @@ function ImageDisplay({ element, box }) {
         ref={ref}
         draggable={false}
         src={resource.src}
+        srcSet={calculateSrcSet(resource)}
```

Comment:
Can resouce have an attribute `srcSet` to avoid creating it in multiple places?

Reply:
Done.

Reply:
I had to revert this change. Unfortunately doing it this was wouldn't be retroactive as the resource objects have already been created in the past.